products/bash/configure-apic-v10.sh

```diff
@@ -13,30 +13,35 @@
 #   - Logged into cluster on the OC CLI (https://docs.openshift.com/container-platform/4.4/cli_reference/openshift_cli/getting-started-cli.html)
 #
 # PARAMETERS:
-#   -n : <namespace> (string), Defaults to "cp4i"
-#   -r : <release-name> (string), Defaults to "ademo"
+#   -n : <NAMESPACE> (string), Defaults to "cp4i"
+#   -r : <RELEASE_NAME> (string), Defaults to "ademo"
+#  
 #
 # USAGE:
 #   With defaults values
 #     ./configure-apic-v10.sh
 #
-#   Overriding the namespace and release-name
+#   Overriding the NAMESPACE and release-name
 #     ./configure-apic-v10 -n cp4i-prod -r prod
 
 function usage {
-    echo "Usage: $0 -n <namespace> -r <release-name>"
+    echo "Usage: $0 -n <NAMESPACE> -r <RELEASE_NAME>"
 }
 
 CURRENT_DIR=$(dirname $0)
 
-namespace="cp4i"
-release_name="ademo"
+NAMESPACE="cp4i"
+RELEASE_NAME="ademo"
+ORG_NAME="main-demo"
```

Comment:
Would it be possible to make this more generic by allowing a list of orgs/catalogs to be passed in as parameters and then create all of those passed in?